packages/design-system/src/Documentation/components/CodeBlock.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+import { Tooltip } from "Tooltip";
+import { toast } from "Toast";
+
+type CodeBlockProps = {
+  code?: string;
+};
+
+function CodeBlock({ code }: CodeBlockProps) {
+  return (
+    <Wrapper
+      onClick={() => {
+        code && navigator.clipboard.writeText(code.toString());
+        toast.show("Copied to clipboard", { kind: "success" });
+      }}
+    >
+      <Tooltip content="Click here to copy" mouseEnterDelay={0.7}>
```

Comment:
Really good idea! Might be a better design to show a copy button up front instead of on hover (the reader might not know that they _can_ hover, so that affordance is missing), but if you feel like this is not a priority you don't need to pick it up. Do create a ticket for it and mark it as `good first issue` if you do table it, though.

Reply:
Can you create a task for this?

Reply:
https://github.com/appsmithorg/appsmith/issues/26847
Already been created